docs/DeveloperGuide.adoc

```diff
@@ -767,20 +817,258 @@ Use case ends.
 +
 Use case resumes at step 2.
 
-{More to be added}
+[discrete]
+
+=== Use case: Delete certain fields
+
+*MSS*
+
+1. User requests list of persons with particular name.
+2. AddressBook shows a list of persons.
+3. User requests to select specific contact.
+4. AddressBook selects contact.
+5. User requests to delete particular field(s).
+6. AddressBook clears the listed field(s).
+
+[discrete]
+=== Use case: Add person
+
+*MSS*
+
+1.  User requests to add persons in the specified format.
+2.  AddressBook adds person in the specified format to the list.
+3.  AddressBook acknowledges that the person has been added to the list.
++
+Use case ends.
+
+*Extensions*
+
+[none]
+*2a. The list is empty.
++
+Use case ends.
+
+* 3a. The given index is invalid.
++
+[none]
+** 3a1. AddressBook shows an error message.
++
+Use case resumes at step 2.
+
+* 5a. The given field is blank.
```

Comment:
What happens then after this event? (e.g. "Use case ends" or "Use case resumes at step XX")